### PR TITLE
Added lighthouse public settings

### PR DESCRIPTION
**Pull request by akarasavov**
@yirensum additional context - https://neo4j.atlassian.net/wiki/spaces/Cluster/pages/511606802/Lighthouse+GA+plan

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1693.surge.sh

Updated pages:
- [Configuration settings](https://neo4j-docs-operations-1693.surge.sh/operations-manual/5/configuration/configuration-settings)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1693.surge.sh

Updated pages:
- [Configuration settings](https://neo4j-docs-operations-1693.surge.sh/operations-manual/5/configuration/configuration-settings)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1693.surge.sh

Updated pages:
- [Configuration settings](https://neo4j-docs-operations-1693.surge.sh/operations-manual/5/configuration/configuration-settings)

**Comment by NataliaIvakina** [comment]
In what version do they become visible? 5.22?

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 1]
```diff
@@ -361,12 +361,58 @@ m|+++10m+++
...
 a|A comma-separated list of endpoints that a server should contact in order to discover other cluster members.
 Typically, all members of the cluster, including the current server, should be specified in this list.
+The setting configures the endpoints for Discovery service V2.
 |Valid values
 a|A comma-separated list where each element is a socket address in the format of `hostname:port`, `hostname`, or `:port`.
 |Default value
 m|
 |===
 
+[role=label--enterprise-edition]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:371]:
```suggestion
[role=label--enterprise-edition, label--new-5.22]
```

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 2]
```diff
@@ -361,12 +361,58 @@ m|+++10m+++
...
+a|A comma-separated list of endpoints that a server should contact in order to discover other cluster members.
+Typically, all members of the cluster, including the current server, should be specified in this list.
+The setting configures the endpoints for Discovery service V2.
+|Valid values
+a|A comma-separated list where each element is a socket address in the format of `hostname:port`, `hostname`, or `:port`.
+|Default value
+m|
+|===
+
+[role=label--enterprise-edition]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:390]:
```suggestion
[role=label--enterprise-edition, label--new-5.22]
```

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 3]
```diff
@@ -3725,6 +3771,22 @@ a|A string.
 m|
 |===
 
+[role=label--enterprise-edition]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:3774]:
```suggestion
[role=label--enterprise-edition, label--new-5.22]
```

**Comment by akarasavov** [comment]
> In what version do they become visible? 5.22?

They are already visible

> In what version do they become visible? 5.22?

Yes, you are right. Those settings will become available in 5.22

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1693.surge.sh

Updated pages:
- [Configuration settings](https://neo4j-docs-operations-1693.surge.sh/operations-manual/5/configuration/configuration-settings)

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 4]
```diff
@@ -361,12 +361,58 @@ m|+++10m+++
...
+[[config_dbms.cluster.discovery.v2.endpoints]]
+=== `dbms.cluster.discovery.v2.endpoints`
+
+// label:enterprise-edition[Enterprise Edition]
+
+.dbms.cluster.discovery.v2.endpoints
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|Description
+a|A comma-separated list of endpoints that a server should contact in order to discover other cluster members.
```
Comment (renetapopova) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:381]: This description should be reflected in the codebase as well. They should fully match.

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 5]
```diff
@@ -361,12 +361,58 @@ m|+++10m+++
...
+Typically, all members of the cluster, including the current server, should be specified in this list.
```
Comment (renetapopova) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:382]:
```suggestion
Typically, all cluster members, including the current server, must be specified in this list.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1693.surge.sh

Updated pages:
- [Configuration settings](https://neo4j-docs-operations-1693.surge.sh/operations-manual/5/configuration/configuration-settings)

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 6]
```diff
@@ -361,12 +361,58 @@ m|+++10m+++
 |Description
 a|A comma-separated list of endpoints that a server should contact in order to discover other cluster members.
 Typically, all members of the cluster, including the current server, should be specified in this list.
+The setting configures the endpoints for Discovery service V2.
```
Comment (renetapopova) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:364]: This description should be reflected in the codebase as well. They should fully match.

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 7]
```diff
@@ -361,12 +361,58 @@ m|+++10m+++
 |Description
 a|A comma-separated list of endpoints that a server should contact in order to discover other cluster members.
 Typically, all members of the cluster, including the current server, should be specified in this list.
```
Comment (renetapopova) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:363]:
```suggestion
Typically, all cluster members, including the current server, must be specified in this list.
```

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 8]
```diff
@@ -361,12 +361,58 @@ m|+++10m+++
...
+[[config_dbms.cluster.discovery.version]]
+=== `dbms.cluster.discovery.version`
+
+// label:enterprise-edition[Enterprise Edition]
+
+.dbms.cluster.discovery.version
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|Description
+a|This setting allows you to select which discovery service should be started.
```
Comment (renetapopova) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:400]: same here

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 9]
```diff
@@ -3725,6 +3771,22 @@ a|A string.
...
+[[config_dbms.kubernetes.discovery.v2.service_port_name]]
+=== `dbms.kubernetes.discovery.v2.service_port_name`
+
+// label:enterprise-edition[Enterprise Edition]
+
+.dbms.kubernetes.discovery.v2.service_port_name
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|Description
+a|Service port name for discovery v2 for Kubernetes API.
```
Comment (renetapopova) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:3784]:
```suggestion
a|Service port name for Discovery v2 for Kubernetes API.
```

Comment (renetapopova) [reply modules/ROOT/pages/configuration/configuration-settings.adoc:3784]: Also, the period is missing in the codebase.

**Review by renetapopova — CHANGES_REQUESTED** [review]
The descriptions are very well-written according to our style. However, in the codebase, they are entirely different. I added some comments mainly related to the fact that they should be the same in the codebase.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1693.surge.sh

Updated pages:
- [Configuration settings](https://neo4j-docs-operations-1693.surge.sh/operations-manual/5/configuration/configuration-settings)

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 10]
```diff
@@ -360,13 +360,59 @@ m|+++10m+++
...
 a|A comma-separated list of endpoints that a server should contact in order to discover other cluster members.
+Typically, all cluster members, including the current server, must be specified in this list.
+The setting configures the endpoints for Discovery service V2.
+|Valid values
+a|A comma-separated list where each element is a socket address in the format of `hostname:port`, `hostname`, or `:port`.
+|Default value
+m|
+|===
+
+[role=label--enterprise-edition, label--new-5.22]
```
Comment (recrwplay) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:371]:
```suggestion
[role=label--enterprise-edition label--new-5.22]
```

To specify multiple roles in this format, separate them with a space.

https://docs.asciidoctor.org/asciidoc/latest/attributes/role/#formal-role-syntax-for-blocks

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 11]
```diff
@@ -360,13 +360,59 @@ m|+++10m+++
...
+a|A comma-separated list of endpoints that a server should contact in order to discover other cluster members.
 Typically, all members of the cluster, including the current server, should be specified in this list.
+The setting configures the endpoints for Discovery service V2.
 |Valid values
 a|A comma-separated list where each element is a socket address in the format of `hostname:port`, `hostname`, or `:port`.
 |Default value
 m|
 |===
 
+[role=label--enterprise-edition, label--new-5.22]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:390]:
```suggestion
[role=label--enterprise-edition label--new-5.22]
```

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 12]
```diff
@@ -3725,6 +3771,22 @@ a|A string.
...
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:3774]:
```suggestion
[role=label--enterprise-edition label--new-5.22]
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1693.surge.sh

Updated pages:
- [Configuration settings](https://neo4j-docs-operations-1693.surge.sh/operations-manual/5/configuration/configuration-settings)

File: modules/ROOT/pages/configuration/configuration-settings.adoc (hunk 4 above)
Comment (akarasavov) [reply modules/ROOT/pages/configuration/configuration-settings.adoc:381]: This [PR](https://github.com/neo-technology/neo4j/pull/26075) changes the description in the main codebase to reflect these changes.

File: modules/ROOT/pages/configuration/configuration-settings.adoc (hunk 6 above)
Comment (akarasavov) [reply modules/ROOT/pages/configuration/configuration-settings.adoc:364]: This [PR](https://github.com/neo-technology/neo4j/pull/26075) changes the description in the main codebase to reflect these changes.

File: modules/ROOT/pages/configuration/configuration-settings.adoc (hunk 8 above)
Comment (akarasavov) [reply modules/ROOT/pages/configuration/configuration-settings.adoc:400]: This [PR](https://github.com/neo-technology/neo4j/pull/26075) changes the description in the main codebase to reflect these changes.

Comment (akarasavov) [reply modules/ROOT/pages/configuration/configuration-settings.adoc:400]: This [PR](https://github.com/neo-technology/neo4j/pull/26075) changes the description in the main codebase to reflect these changes.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1693.surge.sh

Updated pages:
- [Configuration settings](https://neo4j-docs-operations-1693.surge.sh/operations-manual/5/configuration/configuration-settings)

**Review by renetapopova — APPROVED** [review]
Thank you, @akarasavov!

**Pull request merged**